src/components/Layout/Sidebar.module.scss

```diff
@@ -105,6 +105,14 @@
   display: block;
 }
 
+.item__nonProduct {
+  background-color: var(--mm-color-sidebar) !important;
```

Comment:
Is `!important` really necessary here?   Also, this styling only appears when the link is active, no?

Reply:
Should I update this to the following?

```
.item__nonProduct .item__active {
  background-color: var(--mm-color-sidebar);
}
```

Reply:
Almost.  It should be `.item__nonProduct.item__active` since it's on the same element.  What you wrote would target `.item__active` if it was a child of `.item__nonProduct`

Reply:
`item_current` is the class to use instead of `item__active`.

Reply:
> I'm not sure I understand the requested change here. Are you saying to change the isItemActive check to isItemCurrent or something else?

Instead of `.item__nonProduct.item__active`, use `.item__nonProduct.item__current`.